03-client-api/references/answer.yml

```diff
@@ -112,6 +112,22 @@ types:
           returns:
             - "[`Concept`](../concept-api/concept?tab=javascript#concept-methods)"
 
+      - method:
+        common: &method-toJSON
+          title: Retrieve a concept map as a JSON
+          description: Produces a concept map as a JSON, where keys are variable names and values are concepts.
```

Comment:
```suggestion
          title: Retrieve a concept map as JSON
          description: Produces a concept map as a JSON object, where keys are variable names and values are concepts.
```

Reply:
Apply these changes throughout the PR. ("as JSON" being a short form, "as a JSON object" being a longer and more expressive form)